h2o-algos/src/test/java/hex/tree/TreeHandlerTest.java

```diff
@@ -60,6 +61,8 @@ public void testSharedTreeSubgraphConversion() {
             assertEquals(sharedTreeSubgraph.nodesArray.size(), treeProperties._thresholds.length);
             assertEquals(sharedTreeSubgraph.nodesArray.size(), treeProperties._features.length);
             assertEquals(sharedTreeSubgraph.nodesArray.size(), treeProperties._nas.length);
+            final Pattern rootNodeSplitColPattern = Pattern.compile(".* and splits on column '.+'.*");
```

Comment:
Actually tests the split column has some name. Put it here, because I did not want to train yet another model and prolong our test pipeline just for a simple check. This test is perfectly fine for this purpose.